README.md

```diff
@@ -115,6 +115,7 @@ is not needed to install packages with these frameworks:
 | Sylius       | `sylius-theme`
 | symfony1     | **`symfony1-plugin`**
 | TAO          | `tao-extension`
+| TastyIgniter | **`igniter-extension`<br>`igniter-theme`**
```

Comment:
I think, that we should use the full name for types to avoid confusion: `tastyigniter-extension` and `tastyigniter-theme`. Just as it is in CodeIgniter. Can you fix it? Also need to rename `IgniterInstaller` and `IgniterInstallerTest` classes.

Reply:
Done